app/src/main/java/org/mifos/mobilebanking/ui/fragments/HomeOldFragment.java

```diff
@@ -205,6 +205,8 @@ public void onRefresh() {
     }
 
     private void loadClientData() {
+        tvSavingTotalAmount.setText("N/A");
```

Comment:
@jaindiv26 Don't set it to `NA` which means something else, either error message should be consistent or try a hack. The hack here will be to store total saving and total loan amount in shared preference.